docs/reference/metricbeat/metricbeat-metricset-system-ntp.md

```diff
@@ -0,0 +1,62 @@
+---
+mapped_pages:
+  - https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-metricset-system-ntp.html
+---
+
+% This file is generated! See scripts/docs_collector.py
+
+# System ntp metricset [metricbeat-metricset-system-ntp]
+
+The System `ntp` metricset provides Network Time Protocol (NTP) metrics.
+
+This metricset is available on:
+
+* FreeBSD
+* Linux
+* macOS
+* OpenBSD
+* Windows
+
+## Configuration [_configuration_5]
+
+**`ntp.servers`**
+: Remote NTP server addresses of the form "host", "host:port", "host%zone:port", "[host]:port" or "[host%zone]:port". The server may contain an IPv4, IPv6 or domain name address. When specifying both a port and an IPv6 address, one of the bracket formats must be used. If no port is included, NTP default port 123 is used. If multiple servers are specified, metrics are reported separately for each. Defaults to ["pool.ntp.org"].
```

Comment:
The extra spaces after `:` will fix the formatting of the [definition list](https://elastic.github.io/docs-builder/syntax/definition-lists/).

```suggestion
**`ntp.servers`**
:   Remote NTP server addresses of the form "host", "host:port", "host%zone:port", "[host]:port" or "[host%zone]:port". The server may contain an IPv4, IPv6, or domain name address. When specifying both a port and an IPv6 address, one of the bracket formats must be used. If no port is included, NTP default port 123 is used. If multiple servers are specified, metrics are reported separately for each. Defaults to ["pool.ntp.org"].
```